### ISSUE TITLE
Reformatting /* */ as // isn't always desirable

### ISSUE THREAD

**Issue by jdm** [rust-lang-nursery/rustfmt#274]
```diff
-            let unwrapped_obj = UnwrapObject(obj, /* stopAtOuter = */ 0);
+            let unwrapped_obj = UnwrapObject(obj,
+                                             // stopAtOuter =
+                                             0);
```

**Comment by nrc** [comment]
@jdm how often do you do this in Servo? I'm wondering whether we should try to fix this (which would mean adding an option, I don't think there is a general fix that would work here). If it is rare then it could be the sort of thing that is best addressed by `#[rustfmt_skip]`

**Comment by jdm** [comment]
It looks like there are only two instances of it in Servo.

**Comment by nrc** [comment]
Ok, then it is probably something I'd recommend we don't fix. I'll leave the issue open in case someone sees a way to fix it nicely.

**Comment by Stebalien** [comment]
I've configured my editor to auto-reformat my files when I save them which means that any code I've commented out gets reformatted:

```rust
/*
struct Test;

impl Test {
    fn test() {
        println!("test");
    }
}
*/
```

To:
```rust
// struct Test;
//
// impl Test {
// fn test() {
// println!("test");
// }
// }
//
```

**Comment by nrc** [comment]
You can set the option `normalise_comments` to false to avoid this. If `wrap_comments` is also false (the default) then rustfmt should avoid this reformatting. Interested to know if it actually does.

Closing this issue because it is fixed by the `normalise_comments` option.

**Comment by Stebalien** [comment]
Any chance this can be disabled by default? I usually use `/* */` comments for reasons beyond style (they're more convenient when commenting/uncommenting code). Most projects us the defaults and the defaults really should be safe.

**Comment by Stebalien** [comment]
At least until #583 and/or #584 get fixed so I can prevent my commented code from getting flushed to the LHS.

**Comment by nrc** [comment]
Hmm, a lot of times this is quite a desirable reformatting (at least for me). I'd be wary of changing the default. I'll try and find some time to fix #583 asap.
